backend/mediaprovider/subsonic/artistiterator.go

```diff
@@ -30,10 +29,13 @@ func (s *subsonicMediaProvider) IterateArtists(sortOrder string) mediaprovider.A
 	case ArtistSortNameAZ:
 		return s.baseArtistIterFromSimpleSortOrder(
 			func(artists []*subsonic.ArtistID3) []*subsonic.ArtistID3 {
-				slices.SortFunc(artists, func(a, b *subsonic.ArtistID3) int {
```

Comment:
You can rebase and switch back to SortFunc now if you want, since we're moving to Go 1.21. But up to you.

Reply:
Rebased, and moved to the newer approach, while still handling accented characters.